stores/src/main/java/com/issoft/training/stores/utils/ConsoleScanner.java

```diff
@@ -0,0 +1,34 @@
+package com.issoft.training.stores.utils;
+
+import com.issoft.training.stores.listCategories.Store;
+
+import java.util.Scanner;
+
+public class ConsoleScanner {
+    private Store store;
+
+    public ConsoleScanner(Store store) {
+        this.store = store;
+    }
+
+    public void scan() {
+        Scanner scanner = new Scanner(System.in);
+        String name;
+
+        do {
+            System.out.println("Enter command : sort, top, quite");
```

Comment:
you should be able to get user inputs until he enters quit, in other words, he should be able to sort and top as many times as he want.